package.json

```diff
@@ -2,31 +2,49 @@
   "name": "@nerdwallet/shepherd",
   "version": "1.0.0",
   "description": "A utility for applying code changes across many repositories",
-  "scripts": {
-    "build": "tsc",
-    "build:watch": "npm run build -- --watch",
-    "lint": "tslint -p . -c tslint.json 'src/**/*.ts'",
-    "fix-lint": "npm run lint -- --fix",
-    "test": "jest --coverage src/"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/NerdWalletOSS/shepherd.git"
-  },
   "keywords": [
     "codemod",
     "codemods",
     "migration"
   ],
-  "author": "Nathan Walters",
-  "license": "Apache-2.0",
+  "homepage": "https://github.com/NerdWalletOSS/shepherd#readme",
   "bugs": {
     "url": "https://github.com/NerdWalletOSS/shepherd/issues"
   },
-  "homepage": "https://github.com/NerdWalletOSS/shepherd#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/NerdWalletOSS/shepherd.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Nathan Walters",
   "bin": {
     "shepherd": "./lib/cli.js"
   },
+  "scripts": {
+    "build": "tsc",
+    "build:watch": "npm run build -- --watch",
+    "fix-lint": "npm run lint -- --fix",
+    "lint": "tslint -p . -c tslint.json 'src/**/*.ts'",
+    "prepublishOnly": "npm run test && npm run build",
+    "test": "jest --coverage src/"
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx",
+      "json",
+      "node"
+    ],
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    }
+  },
+  "files": [
+    "lib"
```

Comment:
What about package.json? Any other files that may be needed?

Reply:
That, `LICENSE`, and `README.md` are automatically included by `npm`, so we should be good!